### PR TITLE
Emphasize the use of a private rpc provider

### PR DESCRIPTION
**Pull request by shelleyolivia**
Given ever decreasing throughput limits, wanted to emphasize the importance of a private rpc provider and that Alchemy was the preferred provider. Pls lmk if you have any questions, thank you :)

### CODE REVIEW

File: src/docs/guides/cex-dev.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ You connect to Optimism the same way you do to Ethereum, by connecting to a JSON
 
 ### Endpoints
 
-[Click here for the Optimism endpoints](../useful-tools/networks.md). You can choose between our public endpoints, which are rate limited, and [endpoints from infrastructure providers](../useful-tools/networks.md#rpc-endpoints).
+[Click here for the Optimism endpoints](../useful-tools/networks.md). You can choose between our public endpoints, which are rate limited, and [endpoints from infrastructure providers](../useful-tools/networks.md#rpc-endpoints). Give rate throughput limits, we recommend using a private rpc provider like [Alchemy](https://www.alchemy.com/layer2/optimism/?a=818c11a8da) for both mainnet and testnet use cases.
```
Comment (smartcontracts) [inline src/docs/guides/cex-dev.md:12]:
```suggestion
[Click here for the Optimism endpoints](../useful-tools/networks.md). You can choose between our public endpoints, which are rate limited, and [endpoints from infrastructure providers](../useful-tools/networks.md#rpc-endpoints). Given rate throughput limits, we recommend using a private rpc provider like [Alchemy](https://www.alchemy.com/layer2/optimism/?a=818c11a8da) for both mainnet and testnet use cases. 
```

Comment (smartcontracts) [reply src/docs/guides/cex-dev.md:12]: Just a minor nitpick

Comment (shelleyolivia) [reply src/docs/guides/cex-dev.md:12]: Amazing, thanks for the catch @smartcontracts! Appreciate it :)

Comment (shelleyolivia) [reply src/docs/guides/cex-dev.md:12]: Pls lmk if you need anything else to commit this PR

Comment (shelleyolivia) [reply src/docs/guides/cex-dev.md:12]: hey @smartcontracts following up on this - anything else you need to commit? I also have a few open PRs if you don't mind taking a look :) Thanks so much!

**Review by Nicca42 — APPROVED** [review]

**Pull request merged**